packages/postgres-database/src/simcore_postgres_database/models/projects_nodes.py

```diff
@@ -48,7 +56,7 @@
     # TIME STAMPS ----
     column_created_datetime(timezone=True),
     column_modified_datetime(timezone=True),
-    sa.PrimaryKeyConstraint("project_uuid", "node_id"),
```

Comment:
Aha! I did not notice this before. So this created a primary key as (project_uuid, node_id) pairs but would not restrict having e.g. same node_id in different projects, as we thought.

Reply:
correct, I would say the better practice is to have one column primary key, and I have just uniquely constraint the project_uuid + node_id combination.